script.tag-generator/main.py

```diff
@@ -0,0 +1,521 @@
+import re, urllib2, urllib, httplib, sys, os, time
+import xbmc 
+import xbmcgui 
+import xbmcaddon
+import json
+try:
+    import simplejson
+    import trakt
+    from trakt import users
+except:
+    if os.name == "nt":
+        slash = "\\"
+    else:
+        slash = "/"
+    sys.path.append(os.path.abspath(os.path.dirname(__file__)+slash+"resources"+slash+"lib"))
```

Comment:
`os.path.join()` is the correct method of platform-independent joining path strings in Python